pkgs/by-name/du/dungeondraft/package.nix

```
@@ -0,0 +1,88 @@
+{ lib
+, stdenv
+, requireFile
+, dpkg
+, xorg
+, libGL
+, udev
+, pulseaudio
+, libkrb5
+, zlib
+, makeWrapper
+, gnome
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dungeondraft";
+  version = "1.1.0.3";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  src = requireFile {
+    name = "Dungeondraft-${version}-Linux64.deb";
+    url = "https://dungeondraft.net/";
+    hash = "sha256-rvBhmAXotx6o9K8DM9ucyVIDMVGn3N50ag7jQ+I/VSE=";
+  };
```

Consider using `stdenv.mkDerivation (finalAttrs` instead of `rec` here, as it's more idiomatic
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "dungeondraft";
  version = "1.1.0.3";

  nativeBuildInputs = [ makeWrapper ];

  src = requireFile {
    name = "Dungeondraft-${finalAttrs.version}-Linux64.deb";
    url = "https://dungeondraft.net/";
    hash = "sha256-rvBhmAXotx6o9K8DM9ucyVIDMVGn3N50ag7jQ+I/VSE=";
  };
```

other attributes would that are currently workng through `rec` would also need to be prefixed with `finalAttrs.`